src/Test.UnitTests.Sarif/Writers/SarifLoggerTests.cs

```diff
@@ -19,12 +19,61 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public class SarifLoggerTests : JsonTests
     {
+
+        private static string GetResourceContents(string resourceName)
+            => ResourceExtractor.GetResourceText($"SarifLogger.{resourceName}");
+
         private readonly ITestOutputHelper output;
 
         public SarifLoggerTests(ITestOutputHelper output)
         {
             this.output = output;
-        }      
+        }
+
+        [Fact]
+        public void SarifLogger_StreamOwnership()
```

Comment:
>SarifLogger_StreamOwnership [](start = 20, length = 27)

Nit: I prefer test names that read in English as subject-verb phrase, for example: `SarifLogger_KeepsWriterOpenWhenRequested`.